src/main/java/edu/kis/powp/jobs2d/drivers/VisitableDriver.java

```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.drivers.visitor.DriverVisitor;
+
+public interface VisitableDriver {
```

Comment:
For what reason VisitableDriver does not extend Job2dDriver?

Reply:
my reason was that it's better to keep these interfaces independent for more flexible and maintainable code architecture.

Reply:
my reason was that it's better to keep these interfaces independently because:

1. visitable driver is specifically dedicated to Visitor pattern in this case, it's single responsible ensures it doesn't inherit unrelated functionalities.
2.  if the visitor logic change, modifications would only confined to the visitable driver, it simplifies future updates and minimises impact on the broader system.